### PR TITLE
Add source sensors values to attributes

### PR DESCRIPTION
**Pull request by Aulos**
## Proposed change
I often want to check what is the "real" temperature/humidity and I have only this sensor on my main dashboard, so I've added source values as additional attributes.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to an this integration)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [x] The code has been formatted using Black (`black --fast custom_components`)

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated to README.md

### CODE REVIEW

File: custom_components/temperature_feels_like/sensor.py [hunk 1]
```diff
@@ -304,9 +313,9 @@ def _get_wind_speed(self, entity_id: Optional[str]) -> Optional[float]:
...
     async def async_update(self):
         """Update sensor state."""
-        temp = self._get_temperature(self._temp)  # °C
-        humd = self._get_humidity(self._humd)  # %
-        wind = self._get_wind_speed(self._wind)  # m/s
+        self._temp_val = temp = self._get_temperature(self._temp)  # °C
+        self._humd_val = humd = self._get_humidity(self._humd)  # %
+        self._wind_val = wind = self._get_wind_speed(self._wind)  # m/s
 
         _LOGGER.debug("Temp: %s °C  Hum: %s %%  Wind: %s m/s", temp, humd, wind)
```
Comment (Limych) [inline custom_components/temperature_feels_like/sensor.py:322]: Your code makes uninitialized variables. Please update also this line..

File: custom_components/temperature_feels_like/sensor.py [hunk 2]
```diff
@@ -148,8 +154,11 @@ def state_attributes(self):
         """Return the state attributes."""
         return {
```
Comment (Limych) [inline custom_components/temperature_feels_like/sensor.py:157]: Please add tests for new attributes.

**Review by Limych — CHANGES_REQUESTED** [review]

File: custom_components/temperature_feels_like/sensor.py (hunk 1 above)
Comment (Aulos) [reply custom_components/temperature_feels_like/sensor.py:322]: I'm not sure what you mean by 'makes uninitialized variables' - `self._temp_val = temp = self._get_temperature(self._temp)` assigns to both `self._temp_val` and `temp` variables. `self._*_val` variables are initialized in `__init__`

Comment (Limych) [reply custom_components/temperature_feels_like/sensor.py:322]: Oh sorry, my mistake.  Was tired and did not notice the second assignments.

**Review by Limych — CHANGES_REQUESTED** [review]
There are some errors in your code. See tests reports.
Please fix it.

**Comment by Limych** [comment]
I'm sorry, there was a bug with LazyState in the production code that interfered with the tests. I've fixed it now, but I can't restart the tests on your PR on the new code. I see you did a force push - do it again, please.

I have a suspicion that the tests you have written will work with an error. But I can't check it yet because of an error with LazyState.

**Comment by Limych** [comment]
Yes, as I thought:

https://github.com/Limych/ha-temperature-feels-like/actions/runs/4145244708/jobs/7169382877
`FAILED tests/test_sensor.py::test_async_setup_platform - assert 5.555555555555555 == 20`

**Review by Limych — CHANGES_REQUESTED** [review]
By the way, since you are changing the logic of the `async_update` method, you need to check for new attributes in the `test_async_update` function as well.

**Comment by Limych** [comment]
Thanks :heart:

**Pull request merged**